source/geod24/Registry.d

```diff
@@ -15,19 +15,16 @@ module geod24.Registry;
 
 import core.sync.mutex;
 import geod24.concurrency;
+import geod24.LocalRest;
 
 /// Ditto
 public shared struct Registry (API)
```

Comment:
So, now we indent to have multiple registries in a heterogeneous network? Like we would have both `Validator`s and 'FullNode`s in the network.

Reply:
Well, we already kind of did: We can have a registry with all nodes as `FullNode`, or just the validators as `Validator`.